go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/frankban/quicktest v1.11.3 // indirect
 	github.com/fsouza/go-dockerclient v1.7.0
 	github.com/go-kit/kit v0.9.0
-	github.com/golang/protobuf v1.3.3
+	github.com/golang/protobuf v1.5.0
```

Comment:
it seems to me that `github.com/hyperledger/fabric-protos-go` should be returned `github.com/golang/protobuf v1.3.3`

Reply:
The main brunch and the tests on it were recently ported to protobuf 1.5.2 and backported to 2.5. It was not backported to 2.4.